packages/react/src/components/Forms/LabelledBox/index.stories.tsx

```diff
@@ -8,16 +8,32 @@ export default {
 } as Meta<typeof LabelledBox>;
 
 const Template: StoryFn<typeof LabelledBox> = (args) => (
-  <div style={{ height: "3.5rem" }}>
+  <div style={{ height: "3.5rem", border: "1px red solid" }}>
```

Comment:
🤔

Reply:
It was on purpose because this component does not have a border, so it's difficult to understand where labels position themselves without the border. 😕

Reply:
Hell, I like that 😎